libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/NavigationMapboxMap.java

```diff
@@ -1355,6 +1361,19 @@ public Builder withRouteClickPadding(float padding) {
       return this;
     }
 
+    /**
+     * Douglas-Peucker simplification tolerance (higher means simpler geometries and faster performance)
+     * for the GeoJsonSources created to display the route line.
+     *
```

Comment:
Could we add the `tolerance` range and what is the default value? @tmpsantos could you help here? I can't see in

```java
  /**
   * Douglas-Peucker simplification tolerance (higher means simpler geometries and faster performance).
   *
   * @param tolerance the tolerance - Defaults to 0.375
   * @return the current instance for chaining
   */
  @NonNull
  public GeoJsonOptions withTolerance(float tolerance) {
```

Reply:
@LukasPaczos @Guardiola31337 in gl-native, default value for tolerance is [`0.375`](https://github.com/mapbox/mapbox-gl-native-internal/blob/internal/include/mbgl/style/sources/geojson_source.hpp#L25), while default value for geojson-vt / js is 3.0

Range, not sure whether we should define that. ε is an [arbitrary distance number](https://en.wikipedia.org/wiki/Ramer%E2%80%93Douglas%E2%80%93Peucker_algorithm).